saleor/order/forms.py

```diff
@@ -49,11 +49,11 @@ def __init__(self, *args, **kwargs):
         self.fields['email'].widget = forms.HiddenInput()
 
 
-class OrderNoteForm(forms.ModelForm):
+class CustomerNoteForm(forms.ModelForm):
```

Comment:
I think the class should be named `OrderNoteForm` instead of `CustomerNoteForm`.